cmd/trade.go

```diff
@@ -500,6 +505,21 @@ func runTradeCmd(options inputs) {
 	botConfig = convertDeprecatedBotConfigValues(l, botConfig)
 	l.Infof("Trading %s:%s for %s:%s\n", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB)
 
+	tracker := metrics.MakeMetricsTracker(
+		version,
+		runtime.GOOS,
+		*options.ui,
+		*options.strategy,
+		float64(botConfig.TickIntervalSeconds),
+		botConfig.TradingExchange,
+		fmt.Sprintf("%s:%s/%s:%s", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB),
+	)
+
+	e := tracker.SendStartupEvent(l)
+	if e != nil {
+		l.Info("could not send amplitude event")
```

Comment:
it's generally a bad practice to eat up errors. if there is a need to eat up errors then it should have a comment next to it explaining why we have eaten up the error.

For now let's throw the wrapped error (see comment in metricsTracker about how to wrap) so the bot crashes if we cannot send the startup event.
Also we should reference the underlying error and say this is the startup error in the wrapped message